src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -171,6 +194,11 @@ internal void Validate()
                 throw new InvalidOperationException($"A non-empty {nameof(this.HubName)} configuration is required.");
             }
 
+            if (!IsInProductionSlotOrTest() && this.isDefaultHubName)
+            {
+                throw new InvalidOperationException($"Task Hub name must be specified to be a unique value when using slots");
```

Comment:
Can you put a TODO here for us to add a link to documentation in this exception? When we make the TODO, it would be a good idea to add a link to a GitHub issue that describes what we need to do to resolve the TODO.